src/sentry/api/event_search.py

```diff
@@ -622,9 +622,7 @@ def visit_text_filter(self, node, children):
     def visit_text_in_filter(self, node, children):
         (negation, search_key, _, search_value) = children
         operator = "IN"
-        search_value = SearchValue(
-            self.process_list(search_value[1], [(_, _, val) for _, _, val in search_value[2]])
```

Comment:
This might have been from when there was an extra token after? not sure

Reply:
Yeah probably a holdover from before your other refactors... a bit surprised that both work though

Reply:
The list generation here was completely a no-op

Reply:
Lol, wtf was I doing... I wonder if I was unpacking at some point